zebra-rpc/src/queue.rs

```diff
@@ -180,6 +180,9 @@ impl Runner {
                 };
 
                 self.queue.insert(tx.clone());
+                if self.receiver.is_empty() {
+                    break;
+                }
```

Comment:
For some reason i was not able to figure out i had to add this as the `rpc_queue_main_loop` and `rpc_queue_receives_all_txs_from_channel` were stuck without it.